images/access_setup/content/bin/setup_kcp.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The pipelines-service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$0")" >/dev/null
+  pwd
+)"
+
+usage() {
+  printf "Usage:
+    %s [options]
+
+Bootstrap a new KCP instance so it can be managed by pipeline as code
+
+Optional arguments:" "${0##*/}" >&2
+usage_args
+printf "
+Example:
+    %s -d -k /path/to/kcp.kubeconfig
+"  "${0##*/}" >&2
+}
+
+bootstrap_kcp() {
+  kcp_name="$(kubectl config view --minify -o json |
+    jq ".clusters[0].cluster.server" |
+    sed -r -e "s@.*://([^.]*).*@\1@").$(
+    kubectl kcp workspace current | cut -d\" -f2 | cut -d: -f2- | tr ':' '.'
+  )"
+  printf "[KCP: %s\n]" "$kcp_name"
+  kubeconfig="$credentials_dir/kcp/$kcp_name.yaml"
+
+  printf "    - Create workspace:\n"
+  org_ws=$(kubectl kcp workspace current | cut -d\" -f2 | cut -d: -f1,2)
+  kubectl kcp workspace use "$org_ws"
+  if ! kubectl kcp workspace use compute >/dev/null 2>&1; then
+    kubectl kcp workspace create compute --enter >/dev/null 2>&1
+  fi
+  kubectl kcp workspace current
+
+  printf "    - Create service account:\n"
+  kubectl apply -k "$SCRIPT_DIR/../manifests"
```

Comment:
If I am not mistaken you are applying the same manifests for kcp and the compute clusters. It only works because access rights have not been tighten. I would prefer to have them separated to avoid making thing that they are aligned in any way.

Reply:
Done